.github/workflows/static_analysis.yml

```diff
@@ -6,22 +6,32 @@ jobs:
     runs-on: ubuntu-latest
     steps:
     - uses: actions/checkout@v2
-    - uses: WillAbides/setup-go-faster@v1.5.0
+    - uses: WillAbides/setup-go-faster@v1.6.0
       with:
         go-version: '1.16.x'     
 
     - name: Get dependencies
       run: |
         sudo apt-get update && sudo apt-get install gcc libgl1-mesa-dev libegl1-mesa-dev libgles2-mesa-dev libx11-dev xorg-dev
-        go install honnef.co/go/tools/cmd/staticcheck@v0.1.2
+        go install golang.org/x/tools/cmd/goimports@latest
+        go install github.com/fzipp/gocyclo/cmd/gocyclo@latest
+        go install golang.org/x/lint/golint@latest
+        go install honnef.co/go/tools/cmd/staticcheck@v0.1.3
+
     - name: Cleanup repository
       run: rm -rf vendor/
 
-    - name: golangci-lint
-      uses: golangci/golangci-lint-action@v2.4.0
-      with:
-        version: v1.37
-        skip-go-installation: true
+    - name: Vet
+      run: go vet -tags ci ./...
+
+    - name: Goimports
+      run: test -z $(goimports -e -d . | tee /dev/stderr)
+
+    - name: Gocyclo
+      run: gocyclo -over 50 .
```

Comment:
I don't know anything about CI, but shouldn't this be 30 to match the current behavior?

Reply:
You are absolutely correct. I copied over the CI from FyneDesk thinking it would be updated 😅